app/assets/javascripts/application.js

```diff
@@ -1 +1,21 @@
-//= require header-footer-only
+/* global $ */
+
+//= require govuk_publishing_components/dependencies
+
+//= require govuk_publishing_components/lib
+//= require govuk_publishing_components/lib/track-click
+
+//= require analytics
+
+//= require govuk_publishing_components/components/cookie-banner
+//= require govuk_publishing_components/components/layout-header
+//= require govuk_publishing_components/components/feedback
+//= require govuk_publishing_components/modules
+
+//= require modules/global-bar
+//= require modules/cross-domain-tracking
```

Comment:
Probably out of the scope of this PR, I'm left wondering if this script is needed, or whether it should be migrated into the gem with the rest of the analytics code?

Reply:
I'd like to leave that to another PR - I've added it to an [issue noting it though][1].

[1]: https://github.com/alphagov/static/issues/2443